odoo/addons/base/i18n/base.pot

```diff
@@ -32804,3 +32804,9 @@ msgstr ""
 #: model:res.country,name:base.ax
 msgid "Åland Islands"
 msgstr ""
+
+#. module: base
+#: model:res.currency,currency_unit_label:base.VND
```

Comment:
@vietphuong-viindoo module `l10n_vn_viin` nhà mình mới cho dịch trường `currency_unit_label`
em cho 1 bạn dịch toàn bộ các tiền tệ theo cách em làm ở module `l10n_vn_viin/i18n_extra/l10n_vn_viin.pot` và  `l10n_vn_viin/i18n_extra/vi_VN.po` nhé
những thuật ngữ trong đó để ghi đè, khi xuất lại file dịch sẽ không bị ảnh hưởng